docs/src/main/asciidoc/_app-dev-guide/assemblies/modules/development/helloworld/pom.xml

```diff
@@ -0,0 +1,140 @@
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.jboss.as.quickstarts</groupId>
+  <artifactId>helloworld</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>war</packaging>
+
+  <name>helloworld Maven Webapp</name>
+
+  <properties>
+	  <version.server>8.0.0.GA-redhat-00008</version.server>
+	  <version.bom.ee>${version.server}</version.bom.ee>
+	  <version.plugin.wildfly>4.2.0.Final</version.plugin.wildfly>
+  	  <maven.compiler.source>11</maven.compiler.source>
+          <maven.compiler.target>11</maven.compiler.target>
+  </properties>
+
+	<repositories>
+	    <repository>                                                                
+		<id>jboss-public-maven-repository</id>
+		<name>JBoss Public Maven Repository</name>
+		<url>https://repository.jboss.org/nexus/content/groups/public/</url>
+		<releases>
+		     <enabled>true</enabled>
+		     <updatePolicy>never</updatePolicy>
+		 </releases>
+		 <snapshots>
+		     <enabled>true</enabled>
+		     <updatePolicy>never</updatePolicy>
+		 </snapshots>
+		     <layout>default</layout>
+	    </repository>
+	    <repository>                                                                
+		<id>redhat-ga-maven-repository</id>
+		<name>Red Hat GA Maven Repository</name>
+		<url>https://maven.repository.redhat.com/ga/</url>
+		<releases>
+		    <enabled>true</enabled>
+		    <updatePolicy>never</updatePolicy>
+		</releases>
+		<snapshots>
+		    <enabled>true</enabled>
+		    <updatePolicy>never</updatePolicy>
+		</snapshots>
+		<layout>default</layout>
+	    </repository>
+	</repositories>
+
+	<pluginRepositories>
+	    <pluginRepository>
+		<id>jboss-public-maven-repository</id>
+		<name>JBoss Public Maven Repository</name>
+		<url>https://repository.jboss.org/nexus/content/groups/public/</url>
+		<releases>
+		    <enabled>true</enabled>
+		</releases>
+		<snapshots>
+		    <enabled>true</enabled>
+		</snapshots>
+	    </pluginRepository>
+	    <pluginRepository>
+		<id>redhat-ga-maven-repository</id>
+		<name>Red Hat GA Maven Repository</name>
+		<url>https://maven.repository.redhat.com/ga/</url>
+		<releases>
+		    <enabled>true</enabled>
+		</releases>
+		<snapshots>
+		        <enabled>true</enabled>
+		</snapshots>
+	    </pluginRepository>
+	</pluginRepositories>
+
+	<dependencyManagement>
+	    <dependencies>
+		<dependency>
+		    <groupId>org.jboss.bom</groupId>                
+		    <artifactId>jboss-eap-ee-with-tools</artifactId>
+		    <version>${version.bom.ee}</version>
+		    <type>pom</type>
+		    <scope>import</scope>
+		</dependency>
+	    </dependencies>
+	</dependencyManagement>
+	<dependencies>
+	  <dependency>                                             
+	    <groupId>jakarta.servlet</groupId>
+	    <artifactId>jakarta.servlet-api</artifactId>
+	    <scope>provided</scope>                                
+	  </dependency>
+	  <dependency>
+	    <groupId>junit</groupId>
+	    <artifactId>junit</artifactId>
+	    <scope>test</scope>
+	  </dependency>
```

Comment:
@emmartins WDYT?

Reply:
IMO the justification is JUnit 4 is effectively dead. It hasn't had a release in nearly 3 years https://junit.org/junit4/. JUnit 5 is now the defacto standard.

Reply:
Taking the devil's side, why don't we then use junit 5 on wildfly, push the version to BOMs, and then use junit 5 everywhere?

Reply:
We have recently decided to use Junit 5, but I think that for this "first" version of the doc we should keep in sync with helloworld QS, which serves as this doc ready to use example, and once we move all QS (including helloworld) to JUnit 5 as planned[1] then we also change this doc. WDYT @jamezp ?
[1] https://wildfly.zulipchat.com/#narrow/stream/174184-wildfly-developers/topic/junit.205

Reply:
@jamezp ?

Reply:
It's not really about time, we should have the doc in sync with QS used, and right now that is limited to helloworld but hopefully that soon starts to be extended (eg [WFLY-18947](https://issues.redhat.com/browse/WFLY-18947) and [WFLY-18948](https://issues.redhat.com/browse/WFLY-18948)). Helloworld currently uses junit 4, so to move to junit 5 here we need to move the QS to junit 5 too, and that I would like to do for all QS at once. In short it will happen, just not a priority right now, compared with merging the first version of this and allowing others to contribute ;-)